suricata/update/main.py

```diff
@@ -357,6 +357,57 @@ def load_dist_rules(files):
                 logger.error("Failed to open %s: %s" % (path, err))
                 sys.exit(1)
 
+def load_classification(suriconf, files):
+    # TODO ASK if there can be overwriting of classification files
```

Comment:
No need to ask to overwrite files in the rule output directory. That is considered to be managed by suricata-update.